sdk/OoyalaSkinSDK/ooyalaSkinCore.js

```diff
@@ -85,6 +85,11 @@ OoyalaSkinCore.prototype.onSocialButtonPress = function(socialType) {
   (results) => {Log.log(results);});
 };
 
+OoyalaSkinCore.prototype.onSocialAlertDismiss = function() {
+  this.skin.setState({alertTitle: ''});
+  this.skin.setState({alertMessage: ''});
```

Comment:
not a big fan of these state names, but i'll let it pass for now because you didn't add it